backend/src/model/dto.rs

```diff
@@ -182,6 +184,21 @@ pub struct Page<T> {
     pub total_pages: i32,
 }
 
+/// A page of loaded results in a timeline.
```

Comment:
Not sure but maybe that sound better/is clearer?
```suggestion
/// A page of results bounded by time.
```